src/components/Canvas.tsx

```diff
@@ -296,31 +296,33 @@ const Canvas: React.FC<CanvasProps> = ({ data, selectedBits }) => {
   };
 
   return (
-    <div className="flex justify-center items-center flex-row md:h-[85%] h-[80%] w-screen px-4 gap-10">
-      <div className="flex justify-center items-center flex-col h-[85%] w-[95%]">
+    <div className="flex justify-center items-start md:h-[85%] h-[80%] w-screen p-2">
```

Comment:
items-start is a default property we can remove that.

Reply:
okay we can remove that

and for your first comment I will discuss it with Sir and I'll let you know.